globals/respond.go

```diff
@@ -0,0 +1,37 @@
+// Package globals implements variables and functions used by all protocol packages
+package globals
+
+import "github.com/PretendoNetwork/nex-go"
+
+// Respond sends the client a given RMC message
+func Respond(packet nex.PacketInterface, message *nex.RMCMessage) {
+	client := packet.Sender()
+
+	var responsePacket nex.PacketInterface
+
+	if packet, ok := packet.(nex.PRUDPPacketInterface); ok {
+		var prudpPacket nex.PRUDPPacketInterface
+
+		if packet.Version() == 1 {
+			prudpPacket, _ = nex.NewPRUDPPacketV1(client.(*nex.PRUDPClient), nil)
+		} else {
+			prudpPacket, _ = nex.NewPRUDPPacketV0(client.(*nex.PRUDPClient), nil)
+		}
+
+		prudpPacket.SetType(nex.DataPacket)
+		prudpPacket.AddFlag(nex.FlagReliable)
```

Comment:
We should only add the reliable flag if the received packet has it aswell, so that we can support unreliable packets

Reply:
Do we know if unreliable packets also require an ACK? Or are they just sent into the void?

Reply:
According to https://github.com/zeroKilo/GROBackendWV/wiki/DATA-Packet , unreliable packets need ACKs too

Reply:
That's pretty unintuitive tbh, I'd assume "unreliable" was just shouting the packet into the void like regular UDP. Weird